ArtifactOntology.ttl

```diff
@@ -595,7 +595,8 @@ cco:Certificate rdf:type owl:Class ;
 ###  http://www.ontologyrepository.com/CommonCoreOntologies/Chart
 cco:Chart rdf:type owl:Class ;
           rdfs:subClassOf cco:Image ;
-          cco:definition "An Image that is designed to represent an Information Content Entity by means of Written Symbols in order to convey that information in a readily understandable format."@en ;
+          cco:definition "An Image that bears a table, lists, diagrams, or graphs."@en ;
```

Comment:
The previous definition was not clear because it did not define 'written symbol' (which are distinguished from non-symbolic semantic glyphs like letters), however I am always hesitant to enumerate types in a definition, especially since table, diagram, and graph are not currently defined. Also, the items listed should be all singular and not mixed.

Reply:
@cameronmore I guess in my mind the question is a) is this a better definition than the present one? Answer: yes. b) Is there a readily available one that is better? Answer: I don't know. Can you think of one? :)

Reply:
See also the parent definition: 

- Image - An Information Bearing Artifact that is designed to bear some specific Information Content Entity that represents some entity.

An IBE cannot bear an ICE; the range of 'is bearer of' is SDC. Also, the "...Information Content Entity that represents some entity." should just be "...Representational Information Content Entity". 

Suggested revision (not sure what work "specific" is doing so left it out): 

- Image* - An Information Bearing Artifact that is designed to carry some Representational Information Content. 

With that in mind, suggested revision to @neilotte's proposal: 

- Chart* - An Image that is designed to carry some Representational Information Content Entity that is prescribed by some canonical visual format. 

In other words, a chart carries representational content in alignment with some standard for visual communication, e.g. diagrams, graphs, maps, etc.

Reply:
@johnbeve offers:

> Image* - An Information Bearing Artifact that is designed to carry some Representational Information Content.". 

This doesn't distinguish an image from, say, an audio recording. Of course, the original definition wasn't doing that either... 

I understand why the current definition just uses 'represents', since the four main ICEs types are all defined as EqCs based on the specific nature of their aboutness relation to some entity. A representational ICE is just an ICE that represents, as opposed to describes or designates or prescribes. Agree that the use of 'bears' needs to be changed and 'specific' is useless.

Much harder to decide: The differentia for an image has do with it representing a thing because it has an isomorphism with it based on(?) derived from(?) using(?) ....(?) light waves is a visual sense. Os, something like that. Tough stuff. 

If you pushed me to decide today: "An Information Bearing Artifact that is designed to carry an Information Content Entity that represents some entity due to maintaining a visual isomorphism".

Reply:
> Chart* - An Image that is designed to carry some Representational Information Content Entity that is prescribed by some canonical visual format.

I like this, but:
1. use of 'prescribed by' means 'canonical visual format' is a Directive ICE, and people will howl that we don't have that in CCO. We'd at least need to add an elucidation that unpacks it with a promise for new additional content some day.
2. I think there needs to be more in the def that reflects that charts are visualizations of underlying data. Something like: "... canonical visual format for representing data". That's gloss, but ... here I go ..

First pass: "An Image that is designed to carry an Information Content Entity that represents a collection of Information Content Entities due to maintaining an isomorphism using some canonical structured visual format."

Reply:
Re Image*: Good catch man. I like shifting to isomorphism talk; how about (to make the isomorphism relata clear): 
- Image** - An Information Bearing Artifact that is designed to carry an Information Content Entity that
represents some entity owing to a visual isomorphism between the artifact and the entity.

Re Chart* and prescribed: Is the concern just that there is not a 'canonical visual format' subclass of Directive ICE? I'd be in favor of adding such a thing in the future (I think this is still independent of the refactoring, i.e. I suspect we'll need it however that goes). 

Re Chart* and restriction to data: I worry there are plausible examples of charts that aren't used to represent data, e.g. @oliviahobai raised the example of an eye chart used in a vision exam. Are, say, organizational charts and flowcharts always used to represent data? Addressing these sorts of examples made me think we should stay broad.

Reply:
> Image** - An Information Bearing Artifact that is designed to carry an Information Content Entity that
represents some entity owing to a visual isomorphism between the artifact and the entity

This is good. Suggest small change for maximal clarity "...between the bearing artifact and.."

> I'd be in favor of adding such a thing in the future (I think this is still independent of the refactoring, i.e. I suspect we'll need it however that goes).

Agree on both counts. Neither a deal breaker. 

> ddressing these sorts of examples made me think we should stay broad.

Yeah, I can see that. I am ok with leaving it broad for now. But, I wonder if 'chart' in some cases is a misappropriation of the term. I am pretty sure an Org chart is still representing data. Charts are interesting in that their purpose seems to be in representing information in a way that makes it more amenable to deriving insight. I think that still maintains the isomorphism criteria. Yes? 

Also relevant: the letters in an eye chart seem to be another example of GDCs that aren't about anything. Perhaps the isomorphism is between GDC rather than just ICE.